tests/translator/input/api_with_custom_base_path.yaml

```diff
@@ -0,0 +1,20 @@
+Globals:
+  Api:
+    Domain:
+      BasePath: foo-bar
```

Comment:
I don't see this in the output files. 🤔

Reply:
And how is this relevant to https://github.com/aws/serverless-application-model/issues/2279#issuecomment-1007055489?

Does something change with `Globals` versus `Properties`?